VideoLocker/res/drawable/selectable_box_overlay.xml

```diff
@@ -1,13 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:state_pressed="true">
-        <shape android:shape="rectangle">
-            <solid android:color="@color/transparent_white_10" />
-        </shape>
-    </item>
-    <item>
-        <shape android:shape="rectangle">
-            <solid android:color="@android:color/transparent" />
-        </shape>
-    </item>
+    <item android:drawable="@color/transparent_white_10" android:state_pressed="true" />
+    <item android:drawable="@android:color/transparent" />
```

Comment:
Nice change. There seem to be also a few other similar `*selectable_*` which could be similarly optimized, e.g. `light_selectable_box_overlay`, `selectable_rounded_box_overlay`.